general/sparkling.md

```diff
@@ -4,38 +4,36 @@
-
-People who share our values, offer new perspectives, will help us grow, and are fun to be around! 
-
-When choosing a new member we care more if we trust a person and we can rely on their judgement, than their skills or how much their contribute. Character fit is important for us. If we invite you to join us, it means that you have 100% of our trust, and not even a shadow of doubt from anyone on the team. Thanks to that, even if you make a mistake, we've got your back. 
-
-Being in this team doesn't necessarily mean many more responsibilities, although you may receive much (much) more emails than you receive now. The amount of work you would like to put into that depends on you. 
+Support Team's role evolved with time: at the beginning, it was a group of people who shaped the way we put processes in place to support sustainability of Django Girls. Today Support Team makes sure Django Girls community grows and supports Awesomeness Ambassador in their responsibilities. 
 
 You can read more about how the Support Team works, what we do, and how we add people in the [About the Support Team](support-team.md) section. 
 
 ## How we work
 
-There is no structure in our team or the way we work. So far it works for us, but it doesn't mean it's gonna continue working this way forever.
+When we started the Support Team, there was no structure in our team or the way we work. Back then it worked for us, but since we hired Awesomeness Ambassador, things slowly started to change. 
 
-There is no a category of tasks that are assigned specificially to one person, unless the person commit themselves to take care of it. It means that you can do everything or nothing, and that's okay. 
+We're currently changing our structure so that everyone in the Support Team has a specific role or area they take care of. 
 
-It's better to ask for forgiveness, than permission. If you're in the team, it means we trust you in making your own decisions and acting before asking. 
+### Well-being
 
-Do you remember when on planes they say that you should first put your oxygen mask, and then help others? We believe in the same rule. Burnout is real, and self care is more important than anything else. We're all in this for the long run, so make sure to prioritize your free time, relax and hobbies. Don't feel bad when you're sipping a drink when everyone else is working -- just enjoy it. That's why there is so many of us in the team, so that someone can enjoy the sun and the show is still running. 
+Do you remember when on planes they say that you should first put your oxygen mask, and then help others? We believe in the same rule. Burnout is real, and self care is more important than anything else. We're all in this for the long run, so make sure to prioritize your free time, relax and hobbies.
+
+To make sure it's easier for every member of the team to take care of their well-being, we've decided to introduce terms
```

Comment:
I wanted to fix that sentence but I am not sure what terms we've decided to introduce 💃